.github/workflows/build.yaml

```diff
@@ -50,7 +37,7 @@ jobs:
         with:
           context: .
           platforms: linux/amd64,linux/arm64
-          push: ${{ startsWith(github.ref, 'refs/tags/') || inputs.publish_release_artifacts == true }}
+          push: false
```

Comment:
Q: should this workflow dump the image as an artifact now that it no longer pushes the image?

Reply:
we can add it if there is a need.